examples/platforms/simulation/platform-simulation.h

```diff
@@ -158,6 +161,25 @@ void platformRadioDeinit(void);
  */
 void platformRadioReceive(otInstance *aInstance, uint8_t *aBuf, uint16_t aBufLength);
 
+/**
+ * This function signals Tx Done to the radio (instead of echo frames).
+ *
+ * @param[in]  aInstance  A pointer to the OpenThread instance.
+ * @param[in]  pktSeq     Packet sequence number to be confirmed as sent.
+ *
+ */
+void platformRadioTxDone(otInstance *aInstance, uint8_t pktSeq);
```

Comment:
is `pktSeq` necessary?

Reply:
This function uses the ```pktSeq``` to confirm that a specific transmitting packet process is done (according to the simulator). The Packet sequence number ensures that the right TxDone signal is synchronized with the simulator.
At the moment, I don't remember if such process could actually cause some wrong signal of another packet that it is not expecting, but I must have seen this possibility and made of it, like a sanity test.